release.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+ORIG_PWD=$PWD
+MOD_VERSION=$(git log | grep Version | head -1 | sed 's/    Version \([0-9]\+\.[0-9]\+\.[0-9]\+\)\.*/v\1/')
+MOD=$(basename $PWD)
+MOD_ZIP=${MOD}-"${MOD_VERSION}".zip
+
+cd /rimworld/1.2/Mods
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)